sagemaker-fundamentals/create-endpoint/container/src/predictor.py

```diff
@@ -0,0 +1,42 @@
+# This is the file that implements a flask server to do inferences. It's the file that you will modify to adapt to your own inference logic.
```

Comment:
This file implements a flask server for inference. You can modify the file to align with your own inference logic.

Reply:
done